### PR TITLE
Add link to company logo in job details page

### PR DESCRIPTION
**Pull request by alibesinci**

### CODE REVIEW

File: src/components/pages/JobDetails.vue [hunk 1]
```diff
@@ -128,13 +128,15 @@ export default {
...
                    itemtype="http://schema.org/Organization"
               >
-                <img
-                  itemprop="logo"
-                  :src="postData.company.logo"
-                  :alt="postData.company.name"
-                  ref="logo"
-                  @error="handleImageError"
-                >
+                <a :href="postData.company.www" target="_blank">
```
Comment (erdoganbulut) [inline src/components/pages/JobDetails.vue]: Nope, as explained in the picture in the Issue;
When the company logo is clicked, it should go to the 'CompanyJobListing' page.

`<router-link>` must be used.

**Comment by erdoganbulut** [comment]
Thanks for your contribution. 🎖 🎉

I added my review notes.

The content of the pullrequests should contain information related to development. If it solves a issue, the issue id will suffice. 

I'm adding; subscribe: #33

File: src/components/pages/JobDetails.vue (hunk 1 above)
Comment (alibesinci) [reply src/components/pages/JobDetails.vue]: Thanks for the feedback, I totally misunderstood the issue.

**Comment by erdoganbulut** [comment]
Fix #32

**Comment by fatihacet** [comment]
Thanks @alibesinci 👍

**Pull request merged**